extensions/2.0/Khronos/KHR_animation_pointer/README.md

```diff
@@ -0,0 +1,251 @@
+# KHR\_animation_pointer
+
+## Contributors
+
+- Alexey Knyazev
+- Ben Houston, Threekit
+- Bryce Hutchings, Microsoft [@brycehutchings](https://twitter.com/brycehutchings)
+- Don McCurdy
+- Ed Mackey, AGI [@emackey](https://github.com/emackey)
+- Felix Herbst, prefrontal cortex [@hybridherbst](https://twitter.com/hybridherbst)
+- Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+- Jamie Marconi, Microsoft [@najadojo](https://twitter.com/najadojo)
+- Joe Herdman, [@jbherdman](https://github.com/jbherdman)
+- Lewis Weaver, Microsoft
+- Marco Hutter
+- Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+Copyright 2024 The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
```

Comment:
Release Candidate?

Reply:
Will update after the WG approves.

Reply:
We should also revive the samples before going to RC.